src/plugins/intel_gpu/src/kernel_selector/cl_kernels/sdpa_micro.cl

```diff
@@ -513,7 +535,7 @@ KERNEL(micro_sdpa)(OPTIONAL_SHAPE_INFO_ARG
                     LSC_LDCC_L1C_L3C);
         }
 #endif
-#if WITH_ATTN_MASK && defined(PREFETCH_MASK)
+#if WITH_ATTN_MASK && defined(PREFETCH_MASK) && !defined(STATIC_SCALAR_ATTN_MASK_VALUE)
```

Comment:
Do we need this change? It seems `STATIC_SCALAR_ATTN_MASK_VALUE` can only be defined when `WITH_ATTN_MASK` is set to 0

Reply:
yes I removed.